[ASM] [DOCS-7629] new ASM Attacker Explorer topic (#22080)
DataDog/documentation: content/en/security/application_security/threats/attackers.md

---
title: Attackers
kind: documentation
---

# Understanding Attackers

ASM classifies IP addresses as
Benign
Suspicious
Flagged

ASM additionally clusters IPs with similar attributes together.

> Benign: Traffic from APM that has not recently sent attack traffic

Reviewer: Should we reformulate like this to be clearer?
```
Benign: IPs that haven't sent any significant amont of attack attempts
```

Author: Updated based on latest draft, which removes this.

Suspicious: A precursor to Flagged IPs. These IPs have met a minimum threshold of attack traffic to be classified as suspicious, but not the threshold for Flagged.

Flagged: IP’s that send large amounts of attack traffic. We recommend blocking Flagged IPs.

Suspicious and Flagged are available as facets. Benign traffic is shown when evaluating the impact of blocking an IP.

{{< img src="security/application_security/threats/attackers/attacker_explorer_default_view.png" alt="Screenshot of the ASM Attacker Explorer"  >}}

# Clusters
{insert blurb on clusters}

# Relationship to Signals and Traces

> Evidence of an attack is visible on the Trace Explorer, which identifies specific payloads that are threats.

Author: Updated based on latest draft, which removes this.

> Signals correlate traces into actionable alerts.

Reviewer: Maybe like this?
```suggestion
Signals aggregate attack attempts into actionable alerts.
```

Author: Updated based on latest draft, which removes this.

> Attack Explorer identifies attackers with persistent threats which may cross multiple Signals.

Reviewer: And then this?
```suggestion
The Attacker Explorer detects persistent threats from attackers that could span across multiple Signals.
```

Author: Updated based on latest draft, which removes this.
